release-notes/10.0/preview/preview2/containers.md

```diff
@@ -2,8 +2,28 @@
 
 .NET 10 Preview 2 includes the following updates for container images:
 
-- [Feature](#feature)
+- [Native AOT images](#native-aot-images)
 
-## Feature
+## Native AOT SDK images
 
-This is about the feature
+Native AOT .NET apps have faster startup time and smaller memory footprints and can run on machines that don't have the .NET runtime installed.
+For .NET 10, we are publishing new .NET SDK image variants that support building [Native AOT](https://learn.microsoft.com/dotnet/core/deploying/native-aot) apps.
+
+The images are intended for two primary scenarios:
+
+- Building container images that contain a Native AOT app, using the SDK image in a multi-stage Dockerfile
+- Building Native AOT .NET executables using the SDK image as a containerized build environment
```

Comment:
I'm having a hard time understanding the difference between these two scenarios. Is there some subtlety I'm missing?

Reply:
The first is for building a container image, and the second is for building binary executables that can be distributed by themselves. I took these bullets pretty much directly from https://github.com/dotnet/dotnet-docker/discussions/6312.

Reply:
Ok, I was reading too much into it.